packages/@okta/vuepress-site/docs/guides/deploy-your-app/java/index.md

```diff
@@ -4,7 +4,7 @@ title: Java
 
 Java applications typically build into a WAR or a JAR for production.
 
-If you deploy your application as a WAR, it's possible you have a context path. If you do, make sure that you add this to your Login redirect URI and your Logout redirect URI for your Okta app.
+If you deploy your application as a WAR, it's possible you have a context path. If you do, make sure that you add this to your Sign-in redirect URI and your Sign-out redirect URI for your Okta app.
```

Comment:
The style guide says to use "ensure" to mean "make sure". However, the style guide doesn't definitively say you can't use "make sure".

Reply:
Changed to:
> If you do, add this path to your Sign-in redirect URI and your Sign-out redirect URI for your Okta app.